specs/interop/token-bridging.md

```diff
@@ -0,0 +1,97 @@
+# Token Bridging
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Interface](#interface)
+- [Implementation](#implementation)
+- [L1 Native Tokens](#l1-native-tokens)
+- [Cross Chain `transferFrom`](#cross-chain-transferfrom)
+- [Factory](#factory)
+- [Upgrading Existing `OptimismMintableERC20Token`s](#upgrading-existing-optimismmintableerc20tokens)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+Unified ERC20 token bridging built on top of the messaging protocol
+can enable tokens to be fungible and movable across the superchain.
+An issue with bridging is that if the security model is not standardized,
+then the bridged assets are not fungible with each other depending on
+which bridge solution was used to facilitate the transfer between
+domains.
+
+## Interface
+
+The `ISuperchainERC20` interface creates a standard way to send tokens
+between domains. In contrast to [xERC20][xerc20] where the user calls
+a bridge that has `mint` and `burn` privileges on the cross chain enabled
+tokens, the user instead calls methods on the token contract directly.
+
+[xerc20]: https://www.xerc20.com/
+
+```solidity
+interface ISuperchainERC20 {
+  function bridgeERC20(uint256 amount, uint256 chainId) external;
+  function bridgeERC20To(address to, uint256 amount, uint256 chainId) external;
```

Comment:
maybe it could be make sense to instead define methods like `transferFrom(address from,..., uint256 chainId)` such that ISuperchainERC20 is seen more clearly as expanding the capabilities of the `ERC20` (in this case, by extending the functions with another argument for `chainId`)

Reply:
This was an earlier approach but then we decided towards a model of being very explicit so that its obvious that this is a bridging interface